docs/getting-started/01_installation.md

```diff
@@ -1,111 +1,108 @@
+SquareOne uses .env files for managing secrets required at build time. You can see an `.env.sample` file in the root. Before installation some secrets are required.
 
 1. Copy the `.env.sample` file and name it `.env`
-2. Fill in the two required plugins keys
+2. Fill in the required secrets
 
 Notes
 * Never commit a .env file to the repository. They are only for storing env secrets and should not be saved to the codebase.
 * Some licensed plugins are required for SquareOne projects (ACF, GravityForms). Removing them is possible, but is not covered here.
 
+### Start the Project
 
-## SquareOne Local
-
-SquareOne Local is a Docker based development environment and is broken into two parts,  Global Containers (DNS, SSL, SQL management for all SquareOne Projects) and the Local containers for this specific project (HTTP, Cache, Search, etc.). We've built in everything you need for a distributable development environment. 
-
-Notes: 
-* Local containers will NOT work if the Global containers are not running.
-* It's recommended you clone a second copy of SquareOne and run and manage your Global containers from there.
-* You can start the global containers from ANY squareOne install, but it's recommended to have a separate squareOne installation for starting/stopping the global containers. This way your SSL certs are all installed in the same installation.
-
-### Configure Dev Domain & SSL
-
-1. Lookup dev domain that is configured in the `/dev/docker/docker-compose.yml` for `VIRTUAL_HOST=`. 
-2. Configure SSL: Run `npm run docker:global:cert DOMAIN.tribe`, where `DOMAIN` is the dev domain.
-3. Copy `local-config-sample.json` to `local-config.json` and set the `proxy` value to your domain 
+1. Goto any squareone project roo and run `so start`
 
 Notes:
-* As noted before, its recommended to have a "global" install of SquareOne and installing every projects certs there. Otherwise, you'll need to start and stop the global containers from each individual project.
-
-### Spin up containers.
-
-1. Run `npm run docker:global:start` (Turns on the global containers. If you are running multiple projects, this only needs to be run once)
-2. Run `npm run docker:start` (Turns on the local project containers. Recommended you start and stop as needed)
-
-Notes: 
-* Local containers will NOT work if the Global containers are not running. 
+* This command will automatically start the Global and Local project containers. For other commands, please see the [SquareOne Global Docker](https://github.com/moderntribe/square1-global-docker) documentation.
 
 ## Build SquareOne
 
-When you spin up the containers for the first time, some of the build process is completed for you. If for any reason you need to run them again, here are the commands:
+When you spin up the containers for the first time, some of the build process are completed for you. If for any reason you need to run them again, here are the commands:
```

Comment:
This whole wording is a bit weird to me. If you just start a project, only the dependencies are built with composer automatically and FE is untouched. 

Maybe we need a little quick start section at the top which would be something like:

1. Install [Square One Global Docker](https://github.com/moderntribe/square1-global-docker#installation)
2. `so create` when starting a brand new project.
3. or, if you just want to play around: `git clone https://github.com/moderntribe/square-one`.
4. cd into the project and run: `so bootstrap`

Perhaps a blind step to clone your existing project if you want to use that.

Reply:
I added a "create new project section" as well that covers this.